apps/omg_status/lib/omg_status/metric/datadog.ex

```diff
@@ -23,4 +23,21 @@ defmodule OMG.Status.Metric.Datadog do
     :test -> use OMG.Status.Metric.Statix
     _ -> use Statix, runtime_config: true
   end
+
+  use GenServer
+  require Logger
+
+  def start, do: GenServer.start(__MODULE__, [], [])
+
+  def init(_opts) do
+    _ = Logger.info("Starting #{inspect(__MODULE__)} and connecting to Datadog.")
+    Process.flag(:trap_exit, true)
+    connect()
```

Comment:
We plug the UDP connection port to a worker process that we can monitor separately.